CodeStarterCamp_Week3/main.swift

```diff
@@ -10,15 +10,13 @@ import Foundation
 
 var misterLee: Person = Person(name: "misterLee", asset: 10000 )
 var missKim: Person = Person(name: "missKim", asset: 0 )
-var yagombucks: CoffeeShop = CoffeeShop()
+var yagombucks: CoffeeShop = CoffeeShop(menu: [.americano: Coffee.americano.price(), .latte: Coffee.latte.price()])
 
 yagombucks.barista = misterLee
-misterLee.buyCoffee(coffeeShop: yagombucks, menus: [Coffee.americano, Coffee.iceTea])
+misterLee.buyCoffee(menu: [Coffee.americano, Coffee.latte], at: yagombucks)
```

Comment:
열거형은 타입추론을 사용해도 좋다고 생각합니다 ㅎㅎ
위 13번 줄 key에는 잘 사용해주셨군요..!

```suggestion
misterLee.buyCoffee(menu: [.americano, .latte], at: yagombucks)
```